mono/mini/local-propagation.c

```diff
@@ -383,10 +383,8 @@ mono_strength_reduction_ins (MonoCompile *cfg, MonoInst *ins, const char **spec)
 			ins->opcode = OP_ICONST;
 			MONO_INST_NULLIFY_SREGS (ins);
 			ins->inst_c0 = 0;
-#if __s390__
```

Comment:
if you do something like this again, in any codebase, move the if the closing brace and have no else. It is much easier to read that way.